src/test/java/mate/academy/springboot/swagger/SwaggerApplicationTests.java

```diff
@@ -1,9 +1,9 @@
 package mate.academy.springboot.swagger;
 
 import org.junit.jupiter.api.Test;
-import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
 
-@SpringBootTest
+@ContextConfiguration
```

Comment:
why did you change it? Can you run it as a test now?

Reply:
yes